core/src/test/java/org/frankframework/receivers/ReceiverTest.java

```diff
@@ -392,6 +393,13 @@ public void run() {
 		);
 	}
 
+	private static void createMessagingSource(PushingJmsListener listener) throws NoSuchFieldException, IllegalAccessException {
```

Comment:
Hoeft niet static te zijn denk ik?

Reply:
Maar mag wel? Deed IntelliJ automatisch toen ik de refactor deed om deze common-code uit veschillende tests in een method te zetten.

Reply:
Ja klopt, dat doet intellij. Maar dat kan je uitzetten met alt-shift-o als je extract method doet. In dit geval zou het geen static moeten zijn, je roept hem nergens static aan.